packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -52,6 +53,13 @@ export class VaAccordion {
       event.detail.currentTarget.parentNode.parentNode.host || // if we are on IE, `.host` won't be there
       event.detail.currentTarget.parentNode.parentNode;
     // Close the other items if this accordion isn't multi-selectable
+    
+    // Usage for slot to provide context to analytics for header and level
+    getSlottedNodes(clickedItem, null).map(
+      (node: HTMLSlotElement) => {
+        this.slotHeader = node.innerHTML
+        this.slotTag = parseInt(node.tagName.toLowerCase().split('')[1])
+    })
```

Comment:
Instead, we could just put the mapped `getSlottedNodes` return values into some local variables. Something like:

```js
const [headerText, headerLevel] = getSlottedNodes(clickedItem, null).map(
    (node: HTMLSlotElement) => {
        const text = node.innerHTML
        const level = parseInt(/*blah blah */)
        return [text, level]
    })
```

(I haven't tested this)

And then use `headerText` and `headerLevel` down in the analytics details instead of the state variables.